twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -9,7 +9,7 @@ load("//scala:scala.bzl",
 def twitter_scrooge():
   native.maven_server(
     name = "twitter_scrooge_maven_server",
-    url = "http://mirror.bazel.build/repo1.maven.org/maven2/",
+    url = "http://repo1.maven.org/maven2/",
```

Comment:
why remove the mirror?

Reply:
It wasn't intentional, I think. Probably a hasty copy&paste.

Reply:
I wonder if you could factor this currently into a second function: `def twitter_scrooge_2_12()` or something. Then we could actually merge changes like this into the mainline and just switch which repositories we set up.